src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -307,20 +307,18 @@ Active dates of the calendar are specified by:
 - `DayType` nodes describing the active days of a week
   + days of the week are listed in `DayType/properties/PropertyOfDay/DaysOfWeek nodes`. 
   + Expected values MUST be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
-- `OperatingPeriod` nodes describing periods (basically a begining date and an end date) referenced in `DayTypeAssignment`
-- `DayTypeAssignment` nodes with 2 possible uses:
-  + Activate or deactivate a specific day on a DayType (with the nodes `IsAvailable` and `Date`)
+- `OperatingPeriod` nodes describing periods (basically a beginning date and an end date) referenced in `DayTypeAssignment`
+- `DayTypeAssignment` nodes with 2 possible uses (they are applied in the order they appear in the file):
+  + Activate or deactivate a specific day on a DayType (with the nodes `IsAvailable` and `Date`); days of the week described in `DayType` takes precedence
```

Comment:
As discussed, activation of a day may be outside the weekdays in `DayType` definition